specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-10-01/consumption.json

```diff
@@ -2702,6 +2702,46 @@
         }
       }
     },
+	 "/providers/Microsoft.Consumption/billingProfiles/{billingProfileId}/discoverTenants": {
+      "get": {
+        "tags": [
+          "discoverTenants"
+        ],
+        "x-ms-examples": {
+          "TenantPropertiesGet": {
+            "$ref": "./examples/DiscoverTenantsGet.json"
+          }
+        },
+        "operationId": "DiscoverTenants_Get",
+        "description": "Gets a Tenant Properties.",
+        "parameters": [
+          {
+            "name": "billingProfileId",
+            "in": "path",
+            "description": "Billing Profile Id.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DiscoverTenant"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },	
```

Comment:
We are moving our new API "discoverTenants" from "Microsoft. Billing" to "Microsoft.Consumption". All the change we made previously remains the same except for different folder and path.  There is no change in the URL pattern we agreed in our previous discussion.

Reply:
There is change in namespace and addition of one extra parameter "billingAccountId".